Backend/sprintone/src/main/java/com/project/omega/controller/DummyController.java

```diff
@@ -0,0 +1,21 @@
+package com.project.omega.controller;
+
+import org.springframework.stereotype.Repository;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+public class DummyController {
```

Comment:
What is the purpose of this?

Reply:
This is can be removed. This was just to check if the controller is reachable or not.,